bundle/config/variable/resolve_job.go

```diff
@@ -16,9 +16,9 @@ func (l resolveJob) Resolve(ctx context.Context, w *databricks.WorkspaceClient)
 	if err != nil {
 		return "", err
 	}
-	return fmt.Sprint(entity.JobId), nil
+	return strconv.FormatInt(entity.JobId, 10), nil
```

Comment:
I think the original has better readability, fwiw.

Reply:
I'm curious why, because it does not accept additional base parameter?

As someone not familiar with this code I find FormatInt option more readable - it only accepts integer so I know entity.JobID is always an integer.

On the other hand Sprint accepts any so it does not tell me what's entity.JobId type is.

Performance-wise also a big difference. With Sprint you first wrap integer to pass it as any then use reflection to find the type and get the integer back.

Reply:
There is also https://pkg.go.dev/strconv#Itoa - I'm not sure perfsprint is not using that, that call has exactly the same signature as Sprint except it enforces the type of the argument.

Reply:
Alright, the type argument missing in `fmt.Sprint` makes sense, thanks.